client/i18n/locales/english/intro.json

```diff
@@ -362,7 +362,7 @@
         "intro": [
           "Functional Programming is a popular approach to software development. In Functional Programming, developers organize code into smaller functions, then combine those functions to build complex programs.",
           "In this spreadsheet application project, you'll learn about parsing and evaluating mathematical expressions, implementing spreadsheet functions, handling cell references, and creating interactive web interfaces. You'll learn how to dynamically update the page based on user input.",
-          "This project will cover concepts like the <code>map()</code> method, <code>find()</code> method, <code>parseInt()</code>, the <code>includes()</code> method."
+          "This project will cover concepts like the <code>map()</code>, <code>find()</code>, <code>parseInt()</code> and <code>includes()</code> methods."
```

Comment:
`parseInt()` is not a method.

Reply:
It’s a method which converts the string into an integer.

Reply:
no, it's a function
methods are those that you use on other values

Reply:
@ilenia-magoni yeah you are right.
Thanks for the correction.